Expecta/Expecta.h

```diff
@@ -12,3 +12,4 @@ FOUNDATION_EXPORT const unsigned char ExpectaVersionString[];
 
 // Enable shorthand by default
 #define expect(...) EXP_expect((__VA_ARGS__))
+#define expect_fail(...) EXP_expect_fail((__VA_ARGS__))
```

Comment:
how about just `fail` or `failure`? `expect_fail` seems a little off compared to the rest of the DSL

(Also, missing newline)